src/qibolab/sweeper.py

```diff
@@ -0,0 +1,10 @@
+class Sweeper:
```

Comment:
I start being repetitive, but I want to make you discover the benefits of incremental typing :)